test/y2storage/proposal_settings_test.rb

```diff
@@ -39,6 +39,55 @@ def stub_partitioning_features(features = {})
 
   let(:initial_partitioning_features) { {} }
 
+  describe "#deep_copy" do
+    subject(:settings) { described_class.new_for_current_product }
+
+    let(:volumes) do
+      [
+        {
+          "mount_point" => "/", "fs_type" => "xfs", "weight" => 60,
+          "desired_size" => "20GiB", "max_size" => "40GiB"
+        },
+        { "mount_point" => "/home", "fs_type" => "xfs", "weight" => 40, "desired_size" => "10GiB" },
+        # This should reuse the existing logical swap
```

Comment:
I don't get this comment (sorry for not mentioning it before). Looks like a copy&paste which doesn't make sense in the scope of this test.

Reply:
A left over from the copy&paster, :man_facepalming: